internal/cmd/certgen.go

```diff
@@ -117,7 +114,7 @@ func outputCertsForKubernetes(ctx context.Context, cli client.Client, cfg *confi
 }
 
 func patchTopologyInjectorWebhook(ctx context.Context, cli client.Client, cfg *config.Server, caBundle []byte) error {
-	if disableTopologyInjector {
```

Comment:
I think we still need the flag, the cfg is not available in the cert ten phase afaik
https://github.com/envoyproxy/gateway/issues/4935

Reply:
Added back in